ecg_data.py

```diff
@@ -1,12 +1,100 @@
-class hrm:
+import numpy
+import pandas
+
+
+class ecg_data:
     'This is a class'
-    def __init__(self, filename, avgT1,avgT2,bradyT,bradyThresh,tachyT,tachyThresh):
-        self.avgT1 = 0.0
-        self.avgT2 = 10.0
 
-    #def instantHr(self):
+    def __init__(
+            self,
+            filename,
+            begin_time=0,
+            end_time=10,
+            bradyT=5,
+            bradyThresh=60,
+            tachyT=5,
+            tachyThresh=100):
+
```

Comment:
Do you not want to map any of the input variables to object attributes?  (I think you do!)  ;)